src/github/pullRequestOverview.ts

```diff
@@ -133,6 +134,31 @@ export class PullRequestOverviewPanel {
 		}
 	}
 
+	private parseReviews(pullRequestModel: PullRequestModel, timelineEvents: TimelineEvent[]): ReviewState[] {
+		const reviewEvents = timelineEvents.filter(isReviewEvent);
+		const reviews: ReviewState[] = [];
+		const seen = new Map<string, boolean>();
+		for (let i = reviewEvents.length -1; i >= 0; i--) {
+			const reviewer = reviewEvents[i].user;
+			if (!seen.get(reviewer.login)) {
```

Comment:
We need to filter out pull request creator as their comments are being labeled as reviews as well.